hub: add hub pr usage for checkout someone created pull-request (#3170)
tldr-pages/tldr: pages/common/hub.md

`hub pull-request --no-edit`

> - Create a new branch with the contents of the pull request and checkout it for review:

Reviewer: How about this?

```suggestion
- Create a new branch with the contents of a pull request and switch to it:
```

Author: Good! thanks.
